doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
I'm probably going to stop after this comment because there are people who are waaaaay more informed than I am about how this works in Node and why. If I keep going without doing some homework, I'll probably only create annoyance and/or confusion. 

@reqshark I used the word "ignore" because that's what's in the docs. I meant nothing precise by it.

@ronkorving I agree that it's surprising that the epipebomb-enabled behavior isn't the default as that seems to be the default everywhere else (judging from the fact that I've never run up against it piping output through `head` from Perl, Python, Ruby, what have you).